### PR TITLE
Update: remove extra margin from pending item

### PR DESCRIPTION
**Pull request by DanDeMicco**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @DanDeMicco has signed the CLA. Thanks for the pull request!

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #422

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/components/ContentSidebar/ActivityFeed/comment/Comment.scss [hunk 1]
```diff
@@ -119,10 +119,6 @@
 
         &.bcs-is-pending {
             opacity: .4;
-
-            .bcs-comment-text {
-                margin-bottom: 25px;
```
Comment (jeremypress) [inline src/components/ContentSidebar/ActivityFeed/comment/Comment.scss]: Was this from BRUI or did we add this?

Comment (DanDeMicco) [reply src/components/ContentSidebar/ActivityFeed/comment/Comment.scss]: Looks like it was added during the dummy data prototype commit

**Pull request merged**